src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -151,16 +151,16 @@ CREATE TABLE public.app_access (
     role_id UUID REFERENCES roles(id) NOT NULL,
     last_access TIMESTAMPTZ,
     active BOOLEAN DEFAULT TRUE,
-    deleted BOOLEAN DEFAULT FALSE,
+    deleted_at TIMESTAMPTZ DEFAULT NULL,
     created_at TIMESTAMPTZ DEFAULT NOW(),
     modified_at TIMESTAMPTZ DEFAULT NOW()
 );
 
 
+
 CREATE TABLE public.capture_sessions (
     id UUID PRIMARY KEY,
     booking_id UUID REFERENCES bookings(id) NOT NULL,
-    parent_recording_id UUID REFERENCES recordings(id) NOT NULL,
```

Comment:
You still need `parent_recording_id` defined here, otherwise you cannot alter the column later to add the reference
```
 parent_recording_id UUID NOT NULL,
```

Reply:
Also might need to be nullable too